src/QuizAtom.tsx

```diff
@@ -1,7 +1,322 @@
+            return incorrectUnchosenLabel;
+        default:
+            return commonLabelStyle;
+    }
+};
+
+const commonTextStyle = css`
+    padding-left: 8px;
+    padding-top: 0;
+    min-height: auto;
+`;
+
+const unchosenText = css`
+    color: ${neutral[7]};
+    ${commonTextStyle};
+`;
+
+const chosenText = css`
+    color: ${neutral[100]};
+    ${commonTextStyle};
+`;
+
+const textStyle = (state: AnswerState) => {
+    switch (state) {
+        case AnswerState.IncorrectChosen:
+            return chosenText;
+        case AnswerState.CorrectChosen:
+            return chosenText;
+        default:
+            return unchosenText;
+    }
+};
+
+const revealTextStyle = (state: AnswerState) => css`
+    ${textStyle(state)}
+    ${textSans.xsmall()}
+`;
+
+const fieldsetStyle = css`
+    margin-bottom: 12px;
+    border: 0px;
+    padding: 0px;
+`;
+
+// Will appear on screen readers
+const visuallyHidden = css`
+    border: 0 !important;
+    clip: rect(0 0 0 0) !important;
+    height: 1px !important;
+    margin: -1px !important;
+    overflow: hidden !important;
+    padding: 0 !important;
+    position: absolute !important;
+    width: 1px !important;
+`;
+
+const iconStyle = css`
+    svg {
+        fill: ${neutral[100]};
+        height: 26px;
+        width: 26px;
+        view-box: '0 0 26px 26px';
+    }
+`;
+
+const fullWidthStyle = css`
+    width: 100%;
+`;
+
+const IconWrapper = (el: JSX.Element) => <div className={iconStyle}>{el}</div>;
+
+export const QuizAtom = ({ id, questions }: QuizAtomType): JSX.Element => (
+    <div data-atom-id={id}>
+        <form>
+            {questions.map((question) => (
+                <Question
+                    key={question.id}
+                    number={questions.indexOf(question) + 1}
+                    {...question}
+                />
+            ))}
+        </form>
+    </div>
+);
+
+export type QuestionProps = {
+    number: number;
+};
+
+export const Question = ({
+    text,
+    imageUrl,
+    answers,
+    number,
+}: QuestionType & QuestionProps): JSX.Element => {
+    const [Chosen, setChosen] = useState<string | undefined>(undefined);
+
+    return (
+        <div
+            className={css`
+                ${body.medium()}
+            `}
+        >
+            <fieldset className={fieldsetStyle}>
+                <div
+                    className={css`
+                        display: flex;
+                    `}
+                >
+                    <div
+                        className={css`
+                            padding-right: 0.5ch;
+                        `}
+                    >
+                        {number + '.'}
+                    </div>
+                    <legend
+                        className={css`
+                            margin-bottom: 12px;
+                        `}
+                    >
+                        {text}
+                    </legend>
+                </div>
```

Comment:
classNames assigned to classes can be useful in places like this for two reasons:

A classname on the outer div can 'name' the block of html so you're able to style with one level deep element styling and have some context:

```
const questionStyle = css`
	display: flex;
	
	> div {
		padding-right: 0.5ch;
	}

	> legend {
		 margin-bottom: 12px;
	}
`
```

I then have a little more context what this part of the component is.

Also, when developing it's useful to have named classNames (not so much in atoms rendering as I am not sure it is turned on!) in DCR at least it adds the className to the end of the class in dev mode (not prod) so it's much easier to find where your element is and where to style.

This is all slightly stylistic but I've found shallow direct-child styling against a contextual className can be the nice mix of devEx and not bloating CSS.